src/__a11y__/a11y-page-object.ts

```diff
@@ -114,3 +115,13 @@ function ariaLevelViolationsFilter(violation: Axe.Result) {
     violation.nodes.every(node => node.html.startsWith('<tr'))
   );
 }
+
+// There is a known issue when multiple app layout instances rendered on the page. Skip them for pages matching this pattern
+function landmarkViolationFilter(violation: Axe.Result, currentUrl: string) {
+  return (
+    !currentUrl.includes('app-layout/multi') ||
+    (violation.id !== 'landmark-main-is-top-level' &&
+      violation.id !== 'landmark-unique' &&
+      violation.id !== 'landmark-no-duplicate-main')
+  );
+}
```

Comment:
These issues are not new, this PR only made them finally visible. Fixing them will be a separate task, discussed it with our a11y squad already